tex/generic/pgf/libraries/shapes/circuits/pgflibraryshapes.gates.logic.IEC.code.tex

```diff
@@ -55,7 +55,13 @@
   %
   % Adjust for the width of the gate symbol.
   %
-  \setbox\pgf@hbox\hbox{{\pgfkeysvalueof{/pgf/#1 gate IEC  symbol}}}% add braces as color-aftergroup-fix TT
+  \setbox\pgf@hbox\hbox{%
+    % wrap in `\pgfinterruptpicture' to restore font, like how `\pgfnode` sets
+    % `\pgfnodeparttextbox`
+    \pgfinterruptpicture
+    {\pgfkeysvalueof{/pgf/#1 gate IEC symbol}}%
+    \endpgfinterruptpicture
+  }% add braces as color-aftergroup-fix TT
```

Comment:
Comment has to go after the correct set of braces:
```suggestion
      {\pgfkeysvalueof{/pgf/#1 gate IEC symbol}}% add braces as color-aftergroup-fix TT
    \endpgfinterruptpicture
  }%
```
Also indent correctly.

Reply:
I should use `[ci skip]`. 😞